make.js

```diff
@@ -8,6 +8,152 @@ var categorizedApis = {};
 var xmlRefDocs = {};
 var propertyReplacements = {};
 
+let testStatusList = new Map([

```

Comment:
>testStatusList [](http://example.com/codeflow?start=4&length=14)

is this setup manually?

Reply:
ya human has judge which stage its at.  perhaps possible to codegen it during the test run pass and then feed it back in this but that's too complex at this point

Reply:
Ya I do think we should judge it manually.  Putting this into make.js seems a bit odd though. Not sure of a better option - maybe adding a separate file (ex TestStatus.json or something) to easily see/update the test statuses?

Reply:
ya, i can do that in next pr